modules/lunamediahbBidAdapter.js

```diff
@@ -67,20 +67,17 @@ export const spec = {
       const placement = {
         placementId: bid.params.placementId,
         bidId: bid.bidId,
+        traffic: bid.params.traffic || BANNER,
```

Comment:
This should be defined with bid.mediaTypes, the 'traffic' param was removed in previous commits and docs were updated. It is fine to leave the default to banner. The tests will need to be updated to remove hardcoding this param as well.

Reply:
That's just my fault, i reverted to much while fixing native